src/gui/guimain.cpp

```diff
@@ -125,23 +125,17 @@ extern "C" void gui_run(void) {
     //Screens::Init(ScreenFactory::Screen<screen_splash_data_t>);
     Screens::Init(screen_initializer, screen_initializer + (sizeof(screen_initializer) / sizeof(screen_initializer[0])));
 
+    //TIMEOUT variable getting value from EEPROM when EEPROM interface is inicialized
+    GuiDefaults::menu_timeout_enabled = (bool)variant_get_ui8(eeprom_get_var(EEVAR_MENU_TIMEOUT));
     //set loop callback (will be called every time inside gui_loop)
     gui_loop_cb = _gui_loop_cb;
-    //int8_t gui_timeout_id;
     while (1) {
         Screens::Access()->Loop();
         // show warning dialog on safety timer expiration
         if (marlin_event_clr(MARLIN_EVT_SafetyTimerExpired)) {
             MsgBoxInfo(_("Heating disabled due to 30 minutes of inactivity."), Responses_Ok);
         }
         gui_loop();
-        /*if (menu_timeout_enabled) {
```

Comment:
pls also remove `screen_close_multiple.cpp` and `screen_close_multiple.h`
since it is rewritten